client/scenes/stream_gui.cpp

```diff
@@ -458,6 +460,13 @@ void scenes::stream::gui_settings(float predicted_display_period)
 		}
 	}
 
+	int bitrate_ = bitrate.load() / 10'000'000;
+	if (ImGui::SliderInt(_S("Bitrate (Mbps)"), &bitrate_, 1, 20, "%d0"))
+	{
+		bitrate.store(bitrate_ * 10'000'000);
+		send_settings_changed_packet(session, network_session.get(), config, bitrate_ * 10'000'000, predicted_display_period);
+	}
+
```

Comment:
Probably a big change, but I think it would be a better interaction to have the bitrate change similar to the foveation center update: use a small tooltip with thumbstick controls.

With the whole GUI we can't see the image center so users can't judge the quality difference.
Ideally we should also show the encode and decode times.